### PR TITLE
CCMSPUI-467 | replace details component

### PR DESCRIPTION
**Pull request by arunkumar461**

### CODE REVIEW

**Review by farrell-m — APPROVED** [review]
Looks good, just remember to bump the snapshot 👍

File: build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 plugins {
 	id 'net.researchgate.release' version '3.0.2'
-	id 'uk.gov.laa.ccms.springboot.laa-ccms-spring-boot-gradle-plugin' version '0.0.12'
+	id 'uk.gov.laa.ccms.springboot.laa-ccms-spring-boot-gradle-plugin' version '0.0.15-d404077-SNAPSHOT'
```
Comment (PhilDigitalJustice) [inline build.gradle:3]: Wait for the plugin update then use the real version

**Review by PhilDigitalJustice — COMMENTED** [review]
1 comment, wait for other PR then amend :)

**Review by PhilDigitalJustice — APPROVED** [review]
LGTM

**Pull request merged**